### ISSUE TITLE
Lines after here strings are treated as here docs

### ISSUE THREAD

**Issue by tterranigma** [atom/language-shellscript#87]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Here strings are supposed to be one line (See the [manual](https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Here-Strings)). However, they are treated as here docs and next lines are highlighted as strings. If you type type the 

### Steps to Reproduce

1. Create a shell script with syntax highlighting on.
2. Type:
```sh
q <<< $f
another command
```

**Expected behavior:** `another command` should be highlighted as a command.

**Actual behavior:** `another command` is highlighted as a string. If you type:
```sh
q <<< $f
another command
$f
```
then string highlighting stops.

**Reproduces how often:** 100%

### Versions
```
Atom    : 1.21.2
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

(node:27931) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
apm  1.18.10
npm  5.5.1
node 8.8.1 x64
atom 1.21.2
python 2.7.14
```
Arch linux latest.

**Comment by georghirn** [comment]
I got the same behavior.

```
Atom    : 1.21.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
```
Version of `language-shellscript` installed: `0.25.3`
Operating system: `Fedora 26, Kernel: 4.13.8-200.fc26.x86_64`

**Comment by 50Wliu** [comment]
Fixed by #83, should be available in Atom 1.23.0-beta0 and above.
